pkg/ctl/scale/nodegroup.go

```diff
@@ -28,7 +29,7 @@ func scaleNodeGroupCmd() *cobra.Command {
 
 	fs.StringVarP(&cfg.ClusterName, "name", "n", "", "EKS cluster name")
 
-	fs.IntVarP(&cfg.Nodes, "nodes", "N", -1, "total number of nodes (scale to this number)")
+	fs.IntVarP(&ng.DesiredCapacity, "nodes", "N", -1, "total number of nodes (scale to this number)")
```

Comment:
Really like the general renaming of struct fields. The consistency is a real help.